ppr-ui/src/interfaces/account-interface.ts

```diff
@@ -0,0 +1,6 @@
+import { AccountModelIF } from '@/interfaces'
+
+// account model
+export interface AccountIF {
+  account: AccountModelIF
+}
```

Comment:
Not really required if you use our stateModel. 
Can apply an interface to the property vs a new interface for a total state.

Reply:
stateModel does not have an account id on the initial load.

<img width="250" alt="Screen Shot 2023-01-25 at 3 37 14 PM" src="https://user-images.githubusercontent.com/2333290/214717649-3a7a7d1a-76d2-416e-9cf5-ee0383ae04b5.png">

Reply:
Sorry, i don't follow the meaning in the context of this comment specifically.
`null` is still a valid type by which to initialize a property.